flow-typed/npm/material-ui_v1.x.x.js

```diff
@@ -795,12 +795,7 @@ declare module 'material-ui/Icon' {
 }
 
 declare module 'material-ui/IconButton/IconButton' {
-  declare type Color =
-    | 'default'
-    | 'inherit'
-    | 'primary'
-    | 'contrast'
-    | 'accent';
+  declare type Color = 'default' | 'inherit' | 'primary' | 'secondary';
```

Comment:
@lfaucon Had to update types from the official API. I guess we will have to keep coming back to this (hating this idea). 
https://material-ui-next.com/api/icon-button/#iconbutton

Reply:
I'm not worried, I don't think we will change that file any time soon